index.html

```diff
@@ -235,15 +283,28 @@ <h5 class="mb">Would you like to know more information on bentos around you?</h5
 				$(this).superslides('animate', 'prev');
 			});
 
+			
+
 		    $(window).scroll(function () {
 		        EasyPeasyParallax();
+		        if ($('#bento-view-info').isOnScreen() && !isBentoViewAnimated) {
+		        	isBentoViewAnimated = true;
+		        	if (isBentoViewDataSet) {
```

Comment:
actually, the cool thing about boolean comparisons here is that we can actually save the use of the `isBentoViewedDataSet` variable and go straight with:

```javascript
if (currentViewed) $('#bento-view-info').countUpAnimate(0, currentViewed);
```

since when Mixpanel API returns us the bento view info, we set it to `currentViewed` variable and it is no longer *false-y*